core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -483,7 +482,7 @@ private[spark] object SecurityManager {
   val SPARK_AUTH_SECRET_CONF: String = "spark.authenticate.secret"
   // This is used to set auth secret to an executor's env variable. It should have the same
   // value as SPARK_AUTH_SECRET_CONF set in SparkConf
-  val ENV_AUTH_SECRET = "_SPARK_AUTH_SECRET"
+  val ENV_AUTH_SECRET = "SPARK_AUTH_SECRET"
```

Comment:
Why this change? The `_` is normally used to indicate something that users should not set directly, and that's the case with this env variable.